src/sentry/api/endpoints/organization_dashboard_details.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import absolute_import
+
+from django.http import Http404
+
+from sentry.api.base import DocSection
+from sentry.api.bases.organization import (
+    OrganizationEndpoint
+)
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.dashboard import DashboardWithWidgetsSerializer as DashboardModelSerializer
+from sentry.models import Dashboard, ObjectStatus
+
+
+class OrganizationDashboardDetailsEndpoint(OrganizationEndpoint):
+
+    doc_section = DocSection.ORGANIZATIONS
+
+    def get(self, request, organization, dashboard_id):
```

Comment:
A common pattern we use is to load the model in a base class. So you could have `OrganizationDashboardEndpoint`, which implements `convert_args`, and that gets the `Dashboard` model. Saves you manually fetching it in any related endpoint.